### ISSUE TITLE
User Min Count Question

### ISSUE THREAD

**Issue by mustfkeskin** [spotify-research/cosernn#3]
Hello

In this paper mention about user activity counts.
"On average, users in the dataset have 220 sessions during the two-month period, and each session consists of 10 tracks on average."

Is there a reason why there are at least 10 tracks per session or active users with 220 sessions are selected. How were these values chosen?

**Comment by lucasmaystre** [comment]
Hi @mustfkeskin , when building our dataset we sampled a set of users that satisfied some activity constraint. I don't recall precisely what the constraint was, but the idea was to avoid having to deal with cold-start situations (these are important but outside of the scope of the paper)

The numbers we report in the paper (220 sessions on average, 10 tracks per session on average) are just empirical averages we observed in the dataset. We didn't explicitly choose them, but they might be linked to the activity criterion we used to sample users.
